src/components/ConnectWallet.tsx

```diff
@@ -66,9 +77,17 @@ function ConnectButton({setPublicToken, setWalletConnected }: ButtonProps): JSX.
     })();
   }, []);
 
+  if (!isConnected) {
```

Comment:
I think a `disconnectwallet` button would be good to have if user wishes to change the address.